src/components/UserProfileTabs/UserProfileTabs.js

```diff
@@ -17,8 +18,11 @@ class UserProfileTabs extends Component {
   }
 
   componentWillMount() {
+    const { user } = this.props;
```

Comment:
This method `componentWillMount` is deprecated in react v16 and above. You should use `componentDidMount ` instead.

Reply:
Okay. Thanks for the feedback
I will fix that and update the PR as soon as possible